example/build_installed/example_source/example_private_kokkos.cpp

```diff
@@ -0,0 +1,28 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <cstdio>
+#include <iostream>
+
```

Comment:
```suggestion
```

Reply:
done in https://github.com/kokkos/kokkos/pull/7582/commits/97081213e09191caa638fbfa87cb56bcaf552cde